internal/fs/wrappers/error_mapping_test.go

```diff
@@ -76,7 +78,17 @@ func (testSuite *ErrorMapping) TestUnAuthenticatedHttpGoogleApiError() {
 	googleApiError := &googleapi.Error{Code: http.StatusUnauthorized}
 	googleApiError.Wrap(fmt.Errorf("UnAuthenticated error"))
 
-	fsErr := errno(googleApiError)
+	fsErr := errno(googleApiError, testSuite.preconditionErr)
 
 	assert.Equal(testSuite.T(), syscall.EACCES, fsErr)
 }
+
+func (testSuite *ErrorMapping) TestFileClobberedError() {
```

Comment:
Let's add test for both cases - when flag is true and false

Reply:
Added